src/common/Error/styled.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+export const Wrapper = styled.div`
+    margin: 0 auto;
+    max-width: 630px;
+    text-align: center;
+    padding: 16px;
+`;
+
+export const DangerImage = styled.img`
+    @media(max-width: ${({theme}) => theme.breakpoints.xsmall}px) {
+        width: 100px;
+    }
```

Comment:
Why are you adding width to @media. The icon with no background is 100px. It does not need to be scaled on smaller screens.

Reply:
will fix it :)